From 6479c904df3725b0a55566ae733c59107cee4737 Mon Sep 17 00:00:00 2001
From: Sunil K Pandey <skpgkp2@gmail.com>
Date: Thu, 1 Apr 2021 15:47:04 -0700
Subject: [PATCH 074/124] x86-64: Fix ifdef indentation in strlen-evex.S

Fix some indentations of ifdef in file strlen-evex.S which are off by 1
and confusing to read.

(cherry picked from commit 595c22ecd8e87a27fd19270ed30fdbae9ad25426)
---
 sysdeps/x86_64/multiarch/strlen-evex.S | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/sysdeps/x86_64/multiarch/strlen-evex.S b/sysdeps/x86_64/multiarch/strlen-evex.S
index cd022509cb175ac0e24d10d2e56b53319afcd4a3..05838190788b2153247e945d211deb840d993f72 100644
--- a/sysdeps/x86_64/multiarch/strlen-evex.S
+++ b/sysdeps/x86_64/multiarch/strlen-evex.S
@@ -276,10 +276,10 @@ L(last_2x_vec):
 	.p2align 4
 L(first_vec_x0_check):
 	tzcntl	%eax, %eax
-# ifdef USE_AS_WCSLEN
+#  ifdef USE_AS_WCSLEN
 	/* NB: Multiply wchar_t count by 4 to get the number of bytes.  */
 	sall	$2, %eax
-# endif
+#  endif
 	/* Check the end of data.  */
 	cmpq	%rax, %rsi
 	jbe	L(max)
@@ -293,10 +293,10 @@ L(first_vec_x0_check):
 	.p2align 4
 L(first_vec_x1_check):
 	tzcntl	%eax, %eax
-# ifdef USE_AS_WCSLEN
+#  ifdef USE_AS_WCSLEN
 	/* NB: Multiply wchar_t count by 4 to get the number of bytes.  */
 	sall	$2, %eax
-# endif
+#  endif
 	/* Check the end of data.  */
 	cmpq	%rax, %rsi
 	jbe	L(max)
@@ -311,10 +311,10 @@ L(first_vec_x1_check):
 	.p2align 4
 L(first_vec_x2_check):
 	tzcntl	%eax, %eax
-# ifdef USE_AS_WCSLEN
+#  ifdef USE_AS_WCSLEN
 	/* NB: Multiply wchar_t count by 4 to get the number of bytes.  */
 	sall	$2, %eax
-# endif
+#  endif
 	/* Check the end of data.  */
 	cmpq	%rax, %rsi
 	jbe	L(max)
@@ -329,10 +329,10 @@ L(first_vec_x2_check):
 	.p2align 4
 L(first_vec_x3_check):
 	tzcntl	%eax, %eax
-# ifdef USE_AS_WCSLEN
+#  ifdef USE_AS_WCSLEN
 	/* NB: Multiply wchar_t count by 4 to get the number of bytes.  */
 	sall	$2, %eax
-# endif
+#  endif
 	/* Check the end of data.  */
 	cmpq	%rax, %rsi
 	jbe	L(max)
-- 
2.41.0

